cfa_azure/batch.py

```diff
@@ -35,8 +35,8 @@ def create_pool(
 
     # Get credentials
     print("Retrieving service principal credentials...")
-    sp_secret = helpers.get_sp_secret(config)
-    sp_credential = helpers.get_sp_credential(sp_secret, config)
+    sp_credential=DefaultAzureCredential()

```

Comment:
we don't want to use DefaultAzureCredential anymore per EDAV since it becomes ambiguous which way is used to authenticate to Azure

Reply:
I replaced it with ManagedIdentityCredential